active_directory_ldap/tests/add_user.json

```diff
@@ -2,13 +2,6 @@
     "body": {
         "action": "add_user",
         "connection": {
-            "host": "",
```

Comment:
We should keep all these keys around so devs can easily try out the plugin by entering input. Same for all the JSON test files.

Reply:
ok it's like I thought having these keys will cause us to fail validation.

Reply:
It should pass validation having empty strings. I tested this with another plugin..

@mhofert-r7 But I do see that a number of the JSON test files are not valid JSON in this plugin.

```
[*] Executing validator JSONValidator
Validator JSONValidator failed!
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/validate.py", line 29, in validate
    v.validate(spec)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/rules/json_validator.py", line 20, in validate
    raise Exception(f"The following test files are not in proper JSON format: {JSONValidator.invalid_files}")
Exception: The following test files are not in proper JSON format: ['delete.json', 'move_object.json', 'modify_groups.json', 'enable_user.json', 'query.json', 'disable_user.json', 'force_password_reset.json', 'reset_password.json']
```

Example test
```
$ jq '.' tests/disable_user.json
parse error: Unfinished JSON term at EOF at line 19, column 1
```

Reply:
We'll need to fix these.